internals/overlord/overlord.go

```diff
@@ -124,6 +124,10 @@ func New(opts *Options) (*Overlord, error) {
 	if !osutil.IsDir(o.pebbleDir) {
 		return nil, fmt.Errorf("directory %q does not exist", o.pebbleDir)
 	}
+	if !osutil.IsWritable(o.pebbleDir) {
+		return nil, fmt.Errorf("no write permission in directory %q", o.pebbleDir)
```

Comment:
Can we make this error message consistent with the other message above? `directory %q not writeable` or similar.